contracts/SpokePool.sol

```diff
@@ -89,6 +89,12 @@ abstract contract SpokePool is
 
     uint256 public constant MAX_TRANSFER_SIZE = 1e36;
 
+    // Note: this needs to be larger than the max transfer size to ensure that all slow fills are fillable, even if
+    // their fees are negative.
+    // It's important that it isn't too large, however, as it should be multipliable by ~2e18 without overflowing.
+    // 1e40 * 2e18 = 2e58 << 2^255 ~= 5e76
+    uint256 public constant SLOW_FILL_MAX_TOKENS_TO_SEND = 1e40;
```

Comment:
This is a good solution. I think we should limit payoutAdjustmentPct based on the diff between `SLOW_FILL_MAX_TOKENS_TO_SEND` and `MAX_TRANSFER_SIZE`